tooling/profiler/src/flamegraph.rs

```diff
@@ -176,29 +176,38 @@ fn find_callsite_labels<'files>(
     files: &'files impl Files<'files, FileId = fm::FileId>,
 ) -> Vec<String> {
     let mut procedure_id = None;
-    let source_locations = debug_symbols.opcode_location(opcode_location).unwrap_or_else(|| {
-        if let (Some(brillig_function_id), Some(brillig_location)) =
-            (brillig_function_id, opcode_location.to_brillig_location())
-        {
-            let procedure_locs = debug_symbols.brillig_procedure_locs.get(&brillig_function_id);
-            if let Some(procedure_locs) = procedure_locs {
-                for (procedure, range) in procedure_locs.iter() {
-                    if brillig_location.0 >= range.0 && brillig_location.0 <= range.1 {
-                        procedure_id = Some(*procedure);
-                        break;
+    let source_locations = match opcode_location {
+        OpcodeLocation::Acir(idx) => {
+            debug_symbols.acir_opcode_location(&AcirOpcodeLocation::new(*idx)).unwrap_or_default()
```

Comment:
```suggestion
            debug_symbols.acir_opcode_location(opcode_location).unwrap_or_default()
```
Could we just do this? Looks a bit cleaner

Reply:
No, one is OpcodeLocation while the other is AcirOpcodeLocation